domain/src/main/java/org/oppia/domain/question/QuestionAssessmentProgressController.kt

```diff
@@ -93,7 +93,7 @@ class QuestionAssessmentProgressController @Inject constructor(private val dataP
    *     recommended that calling code only listen to this result for failures, and instead rely on [getCurrentQuestion]
    *     for observing a successful transition to another state.
    */
-  fun moveToPreviousQuestion(): LiveData<AsyncResult<Any?>> {
+  private fun moveToPreviousQuestion(): LiveData<AsyncResult<Any?>> {
     check(playing) { "Cannot move to the previous question unless an active training session is ongoing" }
     return MutableLiveData(AsyncResult.success<Any?>(null))
   }
```

Comment:
Do not make changes to this files, as these functions are not getting used yet, but will get used soon.

Reply:
Done.